### ISSUE TITLE
Playback randomly pauses at the end of a song

### ISSUE THREAD

**Issue by alex-tee** [mpv-player/mpv#8386]
This issue wasn't happening before. I believe it started occuring after updating my system (and mpv maybe)

### Important Information

```
mpv 0.33.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
FFmpeg library versions:
   libavutil       56.51.100
   libavcodec      58.91.100
   libavformat     58.45.100
   libswscale      5.7.100
   libavfilter     7.85.100
   libswresample   3.7.100
FFmpeg version: 4.3.1
```
GuixSD, GNOME Shell 3.34.2, GTK 3.24.23

### Reproduction steps

Play files in a directory

### Expected behavior

Playback to continue

### Actual behavior

After a few songs playback will be stuck

### Log file

see attached
[mpv_log.txt](https://github.com/mpv-player/mpv/files/5673414/mpv_log.txt)

**Comment by CounterPillow** [comment]
The log file you've attached does not look to be a complete log file as generated by `--log-file`, so we don't even know what config you're using.

**Comment by alex-tee** [comment]
yeah sorry I ran it with `-v -v` and copied most of it till the end. don't think the previous part of the log is relevant here anyway

**Comment by alex-tee** [comment]
this is still broken btw as of 0.33.0 from archlinux. I can't believe no one else uses mpv with JACK. it happens on any distro if you try to play files in a directory, it just stops playback after a couple of files

```
mpv 0.33.0-dirty Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
FFmpeg library versions:
   libavutil       56.51.100
   libavcodec      58.91.100
   libavformat     58.45.100
   libswscale      5.7.100
   libavfilter     7.85.100
   libswresample   3.7.100
FFmpeg version: n4.3.1
```

**Comment by serprex** [comment]
I'm experiencing this on Windows too

**Comment by apa420** [comment]
Still broken for 0.33.1-2 on arch linux with jack.

**Comment by philipl** [comment]
This might have been fixed by #10483. Please try it out.

**Comment by Dudemanguy** [comment]
This was *probably* fixed by the linked PR. If not, feel free to reopen.
